python_modules/dagster/dagster/_core/workspace/context.py

```diff
@@ -671,6 +672,31 @@ def workspace_load_target(self) -> Optional[WorkspaceLoadTarget]:
     def _origins(self) -> Sequence[CodeLocationOrigin]:
         return self._workspace_load_target.create_origins() if self._workspace_load_target else []
 
+    @property
+    def code_server_specs(self) -> Sequence[Mapping[str, Mapping[str, Any]]]:
```

Comment:
copying over my comment from the previous PR: this is a little intense for a property since it calls methods on the registry and stuff - maybe get_code_server_specs()?